doc/api/errors.md

```diff
@@ -1330,6 +1330,11 @@ The `REPL` module was unable parse data from the REPL history file.
 
 An attempt was made to `require()` an [ES6 module][].
 
+<a id="ERR_SCRIPT_EXECUTION_INTERRUPTED"></a>
+### ERR_SCRIPT_EXECUTION_INTERRUPTED
+
+Script execution was interrupted by `SIGINT` (Ctrl+C).
```

Comment:
I kind of agree with @Trott that this is not exactly accurate. Maybe here we should say `(for example, when Ctrl+C is pressed)` instead